FirebaseCore/Sources/FIRLogger.m

```diff
@@ -28,6 +28,7 @@
 FIRLoggerService kFIRLoggerMLKit = @"[Firebase/MLKit]";
 FIRLoggerService kFIRLoggerPerf = @"[Firebase/Performance]";
 FIRLoggerService kFIRLoggerRemoteConfig = @"[Firebase/RemoteConfig]";
```

Comment:
Delete. See comment above. FirebaseCore should not know about its clients :)

Reply:
Done.